sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -249,7 +247,6 @@ static void CleanupThread()
 
       if (connectionList.empty())
       {
-        Log::Write(Logger::Level::Verbose, "Clean pool - remove index " + index->first);
```

Comment:
Unrelated nit: Should we add some logging around this branch, for `m_cleanThread.join();` (especially now that we are removing the logging within the clean up method)?

https://github.com/Azure/azure-sdk-for-cpp/blob/a8fe6038790859fa2f9243020f4b94070354b39c/sdk/core/azure-core/src/http/curl/curl.cpp#L1475-L1480

Reply:
We don't need it for now.